contrib/externs/jquery-1.9.js

```diff
@@ -645,11 +651,20 @@ jQuery.Event.prototype.isImmediatePropagationStopped = function() {};
 jQuery.Event.prototype.isPropagationStopped = function() {};
 
 /** @type {boolean} */
+jQuery.Event.prototype.isTrigger;
+
+/** @type {number} */
+jQuery.Event.prototype.keyCode;
+
+/** @type {boolean} */
 jQuery.Event.prototype.metaKey;
 
 /** @type {string} */
 jQuery.Event.prototype.namespace;
 
+/** @type {RegExp} */
+jQuery.Event.prototype.namespace_re;
```

Comment:
This is another non-documented property. If you feel like it should be included, then perhaps a PR to api.jquery.com to document the property first is in order. Feel free to cross link to this PR.